onebusaway-android/src/main/java/org/onebusaway/android/ui/ReportProblemFragmentBase.java

```diff
@@ -55,6 +55,13 @@
      */
     GoogleApiClient mGoogleApiClient;
 
+    /**
```

Comment:
@barbeau In addition, I thought this method's implementation can vary in the future. In other words, if someone wants to implement another ```Report*ProblemFragment``` by extending  ```ReportProblemFragmentBase```, they may want to implement different validation method. Hence, I tried to use abstraction on this method. Your solution is going to solve the duplicate code in subclasses, but it might limit the future usage. (Or maybe I am over thinking :))